### PR TITLE
LPS-188465 Add default readinessProbe/livenessProbe to the generated LCP.json by the workspace plugin

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for [LPS-188465](https://issues.liferay.com/browse/LPS-188465).

cc @rotty3000 @gamerson

### CODE REVIEW

File: modules/sdk/gradle-plugins-workspace/src/main/resources/com/liferay/gradle/plugins/workspace/task/dependencies/templates/frontend/LCP.json.tpl [hunk 1]
```diff
@@ -13,10 +13,22 @@
...
+		}
+	},
 	"loadBalancer": {
 		"cdn": true,
 		"targetPort": 80
 	},
 	"memory": 50,
-	"scale": 1
+	"scale": 1,
+	"readinessProbe": {
```
[the remark this answers is not in the document]
Comment (drewbrokke) [reply modules/sdk/gradle-plugins-workspace/src/main/resources/com/liferay/gradle/plugins/workspace/task/dependencies/templates/frontend/LCP.json.tpl:28]: good idea 🤦

**Review by gamerson — APPROVED** [review]
LGTM!

**Review by rotty3000 — APPROVED** [review]
LGTM

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**